### PR TITLE
None tag pollinator metrics

### PR DESCRIPTION
**Pull request by gxueatlassian**
**What's in this PR?**
1. Add isPollinator to all metrics (most of them actually)
2. Minor typings, tags array fix blah blah

**Why**
1. We want to better understand customer metrics, and some metrics are quite skewed by the pollinator, as it fire off every 5 minutes.

**Added feature flags**
N/A

**Affected issues**  
N/A

**How has this been tested?**  
Unit test.

**Whats Next?**

### CODE REVIEW

File: src/config/pollinator.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const STAGE_POLLINATOR_JIRA_HOST = "https://fusion-arc-pollinator-staging-app.atlassian.net";
```
Comment (bgvozdev) [inline src/config/pollinator.ts:1]: Given GH4J is an open-source project, I don't think we should hardcore Atlassian's internals. IMHO this should be controlled by a feature flag...

File: src/config/statsd.ts [hunk 2]
```diff
@@ -29,6 +30,42 @@ export const statsd = new StatsD({
...
 
+type ObjectTags = { [key: string]: string };
+type ExtraInfo = {
+	jiraHost?: string
+}
+
+const wrapPollinatorTags = (tags: ObjectTags, extraInfo: ExtraInfo): Tags => {
+	return {
+		...tags,
+		isPollinator: String(isPollinatorSite(extraInfo.jiraHost))
```
Comment (bgvozdev) [inline src/config/statsd.ts:41]: (continuing my thought from my previous comment)

how about
`isTestMetrics: String(isTestSite(extraInfo.jiraHost))`

?

File: src/config/jira-test-site-check.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { envVars } from "config/env";
+const testSites = (envVars.JIRA_TEST_SITES || "").split(",").filter(s => !!s).map(s => s.trim());
+export const isTestJiraHost = (jiraHost: string | undefined) => {
```
Comment (bgvozdev) [inline src/config/jira-test-site-check.ts:3]: tests?

Comment (gxueatlassian) [reply src/config/jira-test-site-check.ts:3]: Added, was struggling with process.env

**Review by bgvozdev — APPROVED** [review]

**Pull request merged**